backend/python_init_script/main.py

```diff
@@ -2,41 +2,186 @@
 # SPDX-FileCopyrightText: 2021 Felix Steinkohl <steinkohl@campus.tu-berlin.de>
 
 import os
+from dataclasses import dataclass
+import pandas as pd
 from appwrite.client import Client
 from appwrite.services.users import Users
 from appwrite.services.teams import Teams
+from appwrite.exception import AppwriteException
+
+
+@dataclass
+class UserData:
+    """
+    Dataclass for specifying user data
+    """
+
+    name: str
+    email: str
+    team: str
+    roles: [str]
+    init_pw: str
```

Comment:
Can you spell out the name as `init_password` or is this a fixed name by convention?

Reply:
fixed.